CSharp/DevVmPowershell/DevVmPsModules/ResizeDatabasesModule.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Helpers;
+using System;
+using System.Management.Automation;
+
+namespace DevVmPsModules
+{
+	[Cmdlet(VerbsCommon.Resize, "Databases")]
+	public class ResizeDatabasesModule : BaseModule
+	{
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 0,
+			HelpMessage = "Name of the Relativity Instance")]
+		public string RelativityInstanceName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 1,
+			HelpMessage = "Username of the Relativity Sql Account")]
+		public string SqlAdminUserName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 2,
+			HelpMessage = "Password of the Relativity Sql Account")]
+		public string SqlAdminPassword { get; set; }
+
+		protected override void ProcessRecordCode()
+		{
+			//Validate Input arguments
+			ValidateInputArguments();
+
+			ISqlHelper sqlHelper = new SqlHelper(RelativityInstanceName, SqlAdminUserName, SqlAdminPassword);
+
+			sqlHelper.CreateOrAlterShrinkDbProc();
+			sqlHelper.RunShrinkDbProc();
+		}
```

Comment:
can we merge CreateOrAlterShrinkDbProc and RunShrinkDbProc in one method. May be we could call CreateOrAlterShrinkDbProc method inside RunShrinkDbProc method.

Reply:
Done, changed the module as well